packages/core/src/processing/generator/generated_data.rs

```diff
@@ -113,11 +138,13 @@ impl GeneratedData {
     /// * `path` - File path to be written
     /// * `delimiter` - Delimiter to use when writing to `path`
     /// * `join_multi_value_columns` - Whether multi value columns should be joined back together or not
+    /// * `long_form` - Pivots column headers and value pairs to key-value row entries.
     pub fn write_synthetic_data(
```

Comment:
I believe this is also called in the python pipeline, in the [generator file:98](https://github.com/microsoft/synthetic-data-showcase/blob/3155c219dc3408f1413b9d84de4c2e1ded7cecc1/packages/python-pipeline/src/generator.py#L96). I think we should add the extra parameter to that call as well.